discord-stubs/abc.pyi

```diff
@@ -27,7 +27,7 @@ class Snowflake(Protocol):
     def created_at(self) -> datetime.datetime: ...
 
 @runtime_checkable
-class User(Protocol):
+class User(Snowflake):
```

Comment:
This (and `PrivateChannel`) needs `Protocol` added to the end of the inheritance list for this class

Reply:
Ah my bad, I forgot that `Protocol` class always needs to be explicitly present. Fixed.

Reply:
It's easy to forget (I do it all the time). Thanks!